packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -10,7 +10,7 @@ meta:
 <ApiLifecycle access="ie" /><br>
 <ApiLifecycle access="Limited GA" />
 
-## Overview
```

Comment:
@chenningzhang-okta are you ok with this rename? We thought it was confusing that the Okta Expressions in OIE page was only focused on User Profile so we thought adding a clearer title for each feature would be better.

Reply:
Yeah agree that an overview of the Okta Expression Language shouldn't just focus on user profile. But I wonder if we should rewrite this section to be more inclusive, instead of simply rename this title. User profile is a sub-section of "Referencing Attributes". I think the following two paragraphs would be better to have an overview of EL.